config/core/configmaps/netcertmanager.yaml

```diff
@@ -0,0 +1,68 @@
+# Copyright 2020 The Knative Authors
```

Comment:
can we use a different file name? It is no longer `net-xxx`. Maybe just `certmanager.yaml`?

Reply:
I am open to that but I don't have a good name for it. `certmanager.yaml` implies we are configuring certmanager itself but what we do is configure our component.

Reply:
I see. But the CM is named `config-certmanager`, so probably still would go with that. In the docs I said "the Knative cert-manager integration".

Reply:
ok done